src/UniqueVectors.jl

```diff
@@ -76,12 +76,14 @@ findlast(p::EqualTo, uv::AbstractUniqueVector) =
 indexin(a::AbstractArray, b::AbstractUniqueVector) =
     [findlast(isequal(elt), b) for elt in a]
 
-function findall(p::Base.Fix2{typeof(in),<:AbstractUniqueVector},
-                 a::Union{Tuple, AbstractArray})
+function findall(p::Base.Fix2{typeof(in),<:AbstractUniqueVector}, a::AbstractArray)
     # The version in Base creates a Set that is unnecessary in our case.  So
     # the override it here.
     [i for (i, ai) in (@static if VERSION >= v"1.1.0-DEV.832" pairs else enumerate end)(a) if p(ai)]
 end
+function findall(p::Base.Fix2{typeof(in),<:AbstractUniqueVector}, a::Tuple)
+    [i for (i, ai) in (@static if VERSION >= v"1.1.0-DEV.832" pairs else enumerate end)(a) if p(ai)]
+end
```

Comment:
Unrelated to the PR, but I got the following error running tests on Julia master:
```
Error During Test at /Users/me/.julia/dev/UniqueVectors/test/runtests.jl:136
  Test threw exception
  Expression: findall((in)(UniqueVector([34, 56, 35, 1, 5, 0])), [1, 2, 34, 0, 5, 56]) == [1, 3, 4, 5, 6]
  MethodError: findall(::Base.Fix2{typeof(in), UniqueVector{Int64}}, ::Vector{Int64}) is ambiguous. Candidates:
    findall(pred::Base.Fix2{typeof(in)}, x::AbstractArray) in Base at array.jl:2340
    findall(testf::Function, A::AbstractArray) in Base at array.jl:2175
    findall(p::Base.Fix2{typeof(in), <:AbstractUniqueVector}, a::Union{Tuple, AbstractArray}) in UniqueVectors at /Users/me/.julia/dev/UniqueVectors/src/UniqueVectors.jl:79
  Possible fix, define
    findall(::Base.Fix2{typeof(in), <:AbstractUniqueVector}, ::AbstractArray)
  Stacktrace:
   [1] top-level scope
     @ ~/.julia/dev/julia/usr/share/julia/stdlib/v1.7/Test/src/Test.jl:434
ERROR: LoadError: There was an error during testing
in expression starting at /Users/me/.julia/dev/UniqueVectors/test/runtests.jl:136
```

Reply:
Thanks for the report!  I'd say this should be removed from the current PR, as it is best to have different/unrelated changes be in different PRs, and #15 now exists with only this change.

Reply:
Sure, will do.